### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: cli/index.js [hunk 1]
```diff
@@ -54,14 +53,13 @@ const cli = meow(`
  * @return {Void}          Write into strout or file and exit
  */
 const render = (text, tfs = [], locale = 'en-us', output) => {
-  const res = (textr({ locale }).use.apply(null, tfs))(text);
+  const res = (textr({ locale }).use(...tfs))(text);
```
Comment (denysdovhan) [inline cli/index.js:56]: Any reason why `apply` better than `...tfs`?

Comment (ghost) [inline cli/index.js:56]: In this case spread operator is cleaner and makes it more ES2015

Comment (denysdovhan) [inline cli/index.js:56]: Oh sorry. I confused diffs. I've thought you removed spread operator.

**Comment by denysdovhan** [comment]
Gonna merge it as soon as I get home.

**Comment by iamstarkov** [comment]
@denysdovhan i can merge it if you want

**Comment by denysdovhan** [comment]
Do you have npm owner permission?

**Comment by iamstarkov** [comment]
@denysdovhan nope, but i can at least merge it here

**Comment by denysdovhan** [comment]
Merged! Published as `0.2.2`.

@iamstarkov @A now you have owner permissions for package. Sorry, guys, I've forgot to add you.

**Pull request merged**